code/modules/medical/cloning.dm

```diff
@@ -229,6 +229,7 @@
 	H.adjustCloneLoss(150) //new damage var so you can't eject a clone early then stab them to abuse the current damage system --NeoFite
 	H.adjustBrainLoss(src.heal_level + 50 + rand(10, 30)) // The rand(10, 30) will come out as extra brain damage
 	H.Paralyse(4)
+	H.silent += 10 //Can't talk for a while
```

Comment:
This could probably also be fixed by setting stat = 1 so that they are unconscious immediately